src/agoradatatools/etl/utils.py

```diff
@@ -299,3 +300,30 @@ def remove_duplicates_keep_order(lst: List[Any]) -> List[Any]:
         if item not in result:
             result.append(item)
     return result
+
+
+def convert_transformation_result_to_dataframe(
+    result: Union[DataFrame, Dict[str, Any], List[Dict[str, Any]], Any],
+    dataset_name: str,
+) -> Union[DataFrame, Dict[str, Any]]:
+    """Convert the result of a custom transformation to a pandas DataFrame.
```

Comment:
Can you add to/edit the comments here, it looks like this function can return either a data frame _or_ a dict. What is the reasoning behind not converting dicts to data frames?

Reply:
Also please include a list of currently-supported input types.

Reply:
Gotcha, that makes perfect sense! Can you just add a brief note in the docstring saying that "dictionaries are not converted to data frames, in order to preserve backward compatibility."

Reply:
Sure. I can add that.